.gitignore

```diff
@@ -1,10 +1,9 @@
-*~
-*.pyc
```

Comment:
Let's leave `*.pyc` in. Context: when you switch branches after running Python scripts on `master`, this litters `*.pyc` files into the directory, and creates noise on the `gh-pages` branch.